src/context/WalletContext.tsx

```diff
@@ -14,7 +16,8 @@ export type TransactionHash = string;
 export type WalletContextType = {
   connected?: WalletInfo;
   connect: (walletOption: WalletOption) => void;
-  getLovelace: () => Promise<any>;
+  /** Returns the lovelace amount in the wallet from the last time `connect` was called */
+  getLovelace: () => any;
```

Comment:
Is it sufficient to only update the wallet balance when connecting? Will the balance shown be incorrect after the user has purchased an NFT? Previously, we were querying the current balance each time the function was called.

In its current state, the function also seems somewhat redundant since the value can be accessed directly from `connected.lovelace`.

Reply:
The returned balance would be incorrect after a user has purchased an NFT; it would only be updated on page load I believe. However from what I can see, the only place where the balance is currently used is in the "buy" popup, so the effect isn't really noticable. This isn't a great solution, but I considered it the quickest working solution for this Seabug version and chose it because I didn't think we wanted to spend the time that better solutions would take. I have some ideas for better solutions though if you think we should work something else out.

And yeah the function isn't necessary, but I figured it would be better to keep the interface consistent and more flexible.